src/Coordination/KeeperSnapshotManager.cpp

```diff
@@ -323,6 +343,7 @@ void KeeperStorageSnapshot::deserialize(SnapshotDeserializationResult & deserial
         readBinary(path, in);
         KeeperStorage::Node node{};
         readNode(node, in, current_version, storage.acl_map);
+
```

Comment:
Not sure if I should add guards for the cases when the snapshot contains some real data in `/keeper` path.
Maybe refuse to read the snapshot and add a setting to let it just ignore and not add to storage?